gcp_variant_transforms/bq_to_vcf.py

```diff
@@ -178,7 +178,7 @@ def _bigquery_to_vcf_shards(
                 | bigquery_to_variant.BigQueryToVariant(annotation_names))
     if known_args.call_names:
       call_names = (p
-                    | transforms.Create(known_args.call_names)
+                    | transforms.Create(known_args.call_names, reshuffle=False)
```

Comment:
Please don't forget to sync before submit, just to avoid confusion in future why this change in included in this PR.

Reply:
Good call, will do.